docs/operating-scylla/nodetool-commands/removenode.rst

```diff
@@ -21,9 +21,14 @@ is removed from the cluster or replaced.
 Prerequisites
 ------------------------
 
-Using ``removenode`` requires at least a quorum of nodes in a cluster to be available. 
-If the quorum is lost, it must be restored before you change the cluster topology. 
-See :doc:`Handling Node Failures </troubleshooting/handling-node-failures>` for details. 
+* Using ``removenode`` requires at least a quorum of nodes in a cluster to be available. 
+  If the quorum is lost, it must be restored before you change the cluster topology. 
+  See :doc:`Handling Node Failures </troubleshooting/handling-node-failures>` for details.
+
+* Make sure that the number of nodes remaining in the DC after you remove a node
+  will be the same or higher than the Replication Factor configured for the keyspace
+  in this DC. If the number of remating nodes is lower than the RF, the removenode
+  request may fail.
```

Comment:
In this case (and in general), replacing the dead node is preferred over removing it.

(Note that we don't allow bootstrapping new nodes while there are DOWN nodes in the system,
so adding a new node before removing the dead node is not supported)

Reply:
Also, for completeness (doesn't need to end up in this doc): reducing the rf, as recommended above for decommission is not a good idea prior to remove node since we don't have the logic yet to drop the replicas specifically on the dead node in response to the rf decrease. Hence I wouldn't recommend it.

See https://github.com/scylladb/scylladb/issues/18822